corpus/definitions.txt

```diff
@@ -858,3 +858,67 @@ class B {
           (class_parameter
             (identifier)
             (type_identifier)))))))
+
+
+
+===================================
+Operator identifiers
+===================================
+
+type ::[+Ab] = scala.collection.immutable.::[Ab]
+val :: = scala.collection.immutable.::
+
+val +: = scala.collection.+:
+val :+ = scala.collection.:+
+
+val test = id.##
+
+---
+
+(compilation_unit
+  (type_definition
+    (type_identifier)
+    (type_parameters
+      (covariant_type_parameter
+        (identifier)))
+    (infix_type
+      (stable_type_identifier
+        (stable_identifier
+          (identifier)
+          (identifier))
+        (type_identifier))
+      (operator_identifier)
+      (generic_type
+        (type_identifier
+          (MISSING _plainid))
```

Comment:
I honestly don't know what this is.

Found https://github.com/tree-sitter/tree-sitter/issues/1043 but not sure I understand both the issue and a way to fix it..